foxi/onnxifi_dummy.c

```diff
@@ -205,3 +209,12 @@ ONNXIFI_PUBLIC ONNXIFI_CHECK_RESULT onnxStatus ONNXIFI_ABI
   onnxReleaseTraceEvents(onnxTraceEventList* traceEvents) {
   return ONNXIFI_STATUS_SUCCESS;
 }
+
+
+ONNXIFI_PUBLIC ONNXIFI_CHECK_RESULT onnxStatus ONNXIFI_ABI
+onnxWaitEventFor(onnxEvent event,
```

Comment:
Do we need to add this timeout value for each inference or we can just use a global flag to set it and just use `onnxWaitEvent` and set event or status to timeout on timing out?

Reply:
I kinda prefer this way since it will allow the c2 to decide how to wait on the event and makes the timeout expectation explicit. Also we need to add the eventStatus thing somewhere so might as well be here.